### ISSUE TITLE
improving, maintenance, etc.

### ISSUE THREAD

**Issue by Delta-Ark** [neurogamedev/NeurosityUE5-NeurogameTemplate#1]
This all looks really interesting; do you think that you are going to support/develop and/or will ue support/develop this tech; it's quite the frontier?

**Comment by neurogamedev** [comment]
Thanks! I am currently trying to update this template into UE 5.1. While the new template looks nicer, the Node.js plugin breaks even harder in this generation and I am frankly considering archiving this Template since it is basically unshippable either way. The geniuses behind the Neurosity tech don't have any real motivation to support game engines and I can only do so much as a Technical Game Designer. Epic Games rejected my application for the Epic Games Grant to be able to really polish this project. So, it's quite lonely here at the frontier of tech...

I am thinking of using Brainflow to connect the Muse 2 to UE5, in my next try. Maybe that will work better? Or maybe Interaxon will get a move on and finally publish their new game ready SDK.

When it comes to videogames, yes, this is the frontier of neurogames. Neurotech support is flimsy to nonexistent and proper standardization is but a feverish dream.

So, in conclusion, not only is this project not supported on any fronts, this might not be the path of least resistance when it comes to neurogames. :-/
